docs/tutorials/dataflows/chatbot.rst

```diff
@@ -0,0 +1,78 @@
+Gitter ML Inference Chatbot
+===========================
+
+This tutorial shows how to use configs in DFFML operations. We'll be implementing
+a Gitter chatbot. Let's take a look at the final result before moving forward.
+
+.. image:: ./data/gitter.gif
+
+Okay, Let's start!!
+We'll be using the Gitter's Streamping API to collect chats, for this we need an
+authorization token from Gitter. Go to https://developer.gitter.im/apps
+and get the personal access token for your chatbot( If you are redirected to the Gitter docs
+from this URL, sign in and try again).
+
+Our dataflow will take a Gitter room URI as input( In https://gitter.im/dffml/community 
+``dffml/community`` is the URI), listens to chats in the room and replies to
+messages which are directed to our bot.
+
+.. note::
+
+    All the code for this example is located under the
+    `examples/dataflow/chatbot <https://github.com/intel/dffml/blob/master/examples/examples/dataflow/chatbot>`_
+    directory of the DFFML source code.
+
+We'll write the operations for this dataflow in
+
+.. literalinclude:: /../examples/dataflow/chatbot/operations.py
```

Comment:
It would be nice if you split out each operation into its own `literalinclude` via the use of `:lines:`